tests/validators/answers/test_number_answer_validator.py

```diff
@@ -208,6 +208,19 @@ def test_invalid_maximum_minimum_value_from_answer_source():
 
     validator.validate_value_in_limits()
 
-    errors = validator.errors
-    assert errors[0]["message"] == validator.MINIMUM_LESS_THAN_LIMIT
-    assert errors[1]["message"] == validator.MAXIMUM_GREATER_THAN_LIMIT
+    expected_errors = [
+        {
+            "message": validator.MINIMUM_LESS_THAN_LIMIT,
+            "value": -9999999999999999,
+            "limit": -999999999999999,
```

Comment:
can this also use MIN_NUMBER / MAX_NUMBER? and the underscores?

Reply:
Updated to use constants and underscores 🖖